script/install/nfs.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# TODO Separate partition to not kill OS. Best separate disk.
+readonly nfs_dir="/var/nfs/general"
+readonly nfs_export_host="*" # TODO Restrict NFS access to some CIDR.
+readonly nfs_server="10.70.26.96"
```

Comment:
This script with create NFS share on the `$nfs_server` under `$nfs_dir` available to all clients within the same network. If we have a separate NAS then we should change the IP to the NAS IP and skip NFS installation. But it works in the Vagrant setups and polyaxon is still happy meaning that we can have persistent storage either way.